configure

```diff
@@ -12110,16 +12110,11 @@ $as_echo "$lt_cv_ld_force_load" >&6; }
       _lt_dar_allow_undefined='$wl-undefined ${wl}suppress' ;;
     darwin1.*)
       _lt_dar_allow_undefined='$wl-flat_namespace $wl-undefined ${wl}suppress' ;;
-    darwin*) # darwin 5.x on
-      # if running on 10.5 or later, the deployment target defaults
-      # to the OS version, if on x86, and 10.4, the deployment
-      # target defaults to 10.4. Don't you love it?
-      case ${MACOSX_DEPLOYMENT_TARGET-10.0},$host in
-	10.0,*86*-darwin8*|10.0,*-darwin[91]*)
-	  _lt_dar_allow_undefined='$wl-undefined ${wl}dynamic_lookup' ;;
-	10.[012][,.]*)
+    darwin*)
+      case ${MACOSX_DEPLOYMENT_TARGET},$host in
+	10.[012],*|,*powerpc*)
 	  _lt_dar_allow_undefined='$wl-flat_namespace $wl-undefined ${wl}suppress' ;;
-	10.*)
+	*)
```

Comment:
Please use autotools 2.69 so that we don't get extra merge conflicts. Can you use the bootstrap script?

Reply:
Should we bump up to 2.71?

Reply:
@Kaustubh-Khedkar : You can check your version by doing

`$ autoconf --version`
 and
`$automake --version`

I get

```
$ automake --version
automake (GNU automake) 1.16.1
Copyright (C) 2018 Free Software Foundation, Inc.
License GPLv2+: GNU GPL version 2 or later <https://gnu.org/licenses/gpl-2.0.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by Tom Tromey <tromey@redhat.com>
       and Alexandre Duret-Lutz <adl@gnu.org>.

```

```
$ autoconf --version
autoconf (GNU Autoconf) 2.69
Copyright (C) 2012 Free Software Foundation, Inc.
License GPLv3+/Autoconf: GNU GPL version 3 or later
<http://gnu.org/licenses/gpl.html>, <http://gnu.org/licenses/exceptions.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by David J. MacKenzie and Akim Demaille.

```

Reply:
Brew does not support 2.71 yet

```
$ brew info automake
automake: stable 1.16.2 (bottled)
Tool for generating GNU Standards-compliant Makefiles
https://www.gnu.org/software/automake/
/usr/local/Cellar/automake/1.16.1_1 (131 files, 3.4MB) *
  Poured from bottle on 2020-03-11 at 20:59:36
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/automake.rb
License: GPL-2.0-or-later
==> Dependencies
Required: autoconf ✔
==> Analytics
install: 66,103 (30 days), 224,537 (90 days), 947,117 (365 days)
install-on-request: 43,484 (30 days), 144,109 (90 days), 603,240 (365 days)
build-error: 0 (30 days)
```

```
$ brew info autoconf
autoconf: stable 2.69 (bottled)
Automatic configure script builder
https://www.gnu.org/software/autoconf
/usr/local/Cellar/autoconf/2.69 (67 files, 3.0MB) *
  Poured from bottle on 2020-03-11 at 20:59:34
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/autoconf.rb
License: GPL-2.0
==> Analytics
install: 110,072 (30 days), 318,196 (90 days), 1,169,418 (365 days)
install-on-request: 17,677 (30 days), 52,773 (90 days), 220,699 (365 days)
build-error: 0 (30 days)
```

Reply:
I don't really see any reason to upgrade as long as things work.

Reply:
Hm, that's weird. Lets fix everything else first and then we'll address autoconf problems at the end. If nothing else works I can just write a patch for you.